docs/topics/crud-mission-database-interaction.adoc

```diff
@@ -1,5 +1,5 @@
 [[common-crud-database-interaction]]
-= Interacting with the CRUD API
+= Interacting with the Relational Database Backend API
```

Comment:
Right, .sorry, now that I look at it, I see that my phrasing is extremely confusing here.
Would it make sense to call the application that provides CRUD functionality  a `CRUD Application`? This would only be done to make the name more specific, so that we don't end up simply calling everything an 'application':)